zenoh/src/net/routing/dispatcher/queries.rs

```diff
@@ -146,7 +146,7 @@ pub(crate) fn undeclare_queryable(
             Some(prefix) => match Resource::get_resource(prefix, expr.suffix.as_ref()) {
                 Some(res) => Some(res),
                 None => {
-                    tracing::error!(
+                    tracing::trace!(
```

Comment:
This one should not occur with proper ACL (expo is not empty). Please revert to error.